components/emoji.rst

```diff
@@ -0,0 +1,125 @@
+The Emoji Component
+===================
+
+    The Emoji component provides access to emoji characters and sequences
+    from the `Unicode CLDR dataset`_, and utilities to manipulate their textual representation.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/emoji
+
+.. include:: /components/require_autoload.rst.inc
+
+
+Transliteration
```

Comment:
Instead of creating two main sections (transliteration and string manipulation) we could divide the contents into the three actual utilities provided by the component:

* Emoji Transliteration
* Emoji Slugger
* Removing Emojis

Reply:
Yes much better, thank you!